runelite-client/src/main/java/net/runelite/client/eventbus/EventBus.java

```diff
@@ -106,7 +106,7 @@ public synchronized void register(@Nonnull final Object object)
 			builder.putAll(subscribers);
 		}
 
-		builder.orderValuesBy(Comparator.comparing(Subscriber::getPriority)
+		builder.orderValuesBy(Comparator.comparing(Subscriber::getPriority).reversed()
```

Comment:
This probably breaks https://github.com/runelite/runelite/commit/9a2872eacfd929cbed22bac1a6bc8835bde7175f